src/main.c

```diff
@@ -1211,7 +1215,7 @@ void thread5_game_logic(s32 arg0) {
     func_800C5CB8();
 loop_3:
```

Comment:
Try while(1) here since there doesn't appear to be a condition for the goto statement.